people/lintong/page.en.md

```diff
@@ -9,9 +9,9 @@ info:
 
 > Lin Tong is a girl who is a bit insecure.
 > She has a mother who she rarely contacts.
-> Dishes like “scrambled eggs with tomatoes” were her favorites.
+> Dishes like “scrambled eggs with tomatoes” were her favourites.
 > Monster Energy was her drink of choice.
-> Personalitywise,
+> Personality wise,
```

Comment:
Personalitywise is indeed a real word.
https://en.wiktionary.org/wiki/personalitywise